ee/session_recordings/playlist_counters/recordings_that_match_playlist_filters.py

```diff
@@ -491,10 +493,16 @@ def count_recordings_that_match_playlist_filters(playlist_id: int) -> None:
 
 
 def enqueue_recordings_that_match_playlist_filters() -> None:
-    base_query = SessionRecordingPlaylist.objects.filter(
-        deleted=False,
-        filters__isnull=False,
-    ).filter(Q(last_counted_at__isnull=True) | Q(last_counted_at__lt=timezone.now() - timedelta(hours=2)))
+    base_query = (
```

Comment:
This change seems to mirror the one from https://github.com/PostHog/posthog/pull/31881/files#diff-06dfa3fce2a028232679339b7511d8c0bf2050e4c295f0febea304070e87e031R496 ?

Reply:
yep, realised I couldn't merge the other one without checking with Raquel (since I still don't grok the digest) and am too excited about this change 🤣

Reply:
(so I split it out)